src/igvfd/schemas/mixins.json

```diff
@@ -26,7 +26,8 @@
             "comment": "Do not submit. A place for the DACC to keep information that does not have a place in the schema.",
             "type": "string",
             "pattern": "^(\\S+(\\s|\\S)*\\S+|\\S)$",
-            "permission": "import_items"
+            "permission": "admin_only",
```

Comment:
why formInput is added here?

Reply:
my mistake I added it on accident when I rebased on https://igvf.atlassian.net/browse/IGVF-1533